internal/cmd/schema-registry/command_cluster_update.go

```diff
@@ -102,35 +102,59 @@ func (c *command) getConfigUpdateRequest(cmd *cobra.Command) (srsdk.ConfigUpdate
 		return updateReq, err
 	}
 
-	var metadataDefaults srsdk.Metadata
-	var metadataOverrides srsdk.Metadata
-	var rulesetDefaults srsdk.RuleSet
-	var rulesetOverrides srsdk.RuleSet
-
-	if err := readPathFlag(cmd, "metadata-defaults", &metadataDefaults); err != nil {
-		return updateReq, err
+	updateReq = srsdk.ConfigUpdateRequest{
```

Comment:
It's more idiomatic to return `srsdk.ConfigUpdateRequest{}, err` above.
```suggestion
	updateReq := srsdk.ConfigUpdateRequest{
```